doc/JSON_INFO.md

```diff
@@ -651,6 +653,10 @@ For information about tools with option to export ASCII art in format ready to b
 | learned_proficiencies       | (_optional_) Array of proficiency ids you gain when installing this CBM, and lose when unintalling
 | installation_requirement    | (_optional_) Requirment id pointing to a requirment defining the tools and componentsnt necessary to install this CBM.
 | vitamin_absorb_mod          | (_optional_) Modifier to vitamin absorbtion, affects all vitamins. (default: `1.0`)
+| block_unins                 | (_optional_) Whether the bionic can be uninstalled or not. (default: `false`)
+| block_unins_msg             | (_optional_) The text shown when trying to uninstall a bionic which cannot be uninstalled. Should be with a possessive pronouns & a nominative one, both replaced with '%s'.
```

Comment:
```suggestion
| cant_uninstal                 | (_optional_) Whether the bionic can be uninstalled or not. (default: `false`)
| cant_uninstal_msg             | (_optional_) The text shown when trying to uninstall a bionic which cannot be uninstalled. Should be with a possessive pronouns & a nominative one, both replaced with '%s'.
```
Although I think using a flag instead of a bool for `block_unins` would be better